### PR TITLE
Calculate workers count per queue. Add kill_group method

### PR DESCRIPTION
**Pull request by hunkom**

### CODE REVIEW

File: arbiter/__init__.py [hunk 1]
```diff
@@ -37,23 +35,22 @@ def _create_task(func):
...
+    def _create_task_from_callable(self, func, name=None, **kwargs):
         name = name if name else f"{func.__name__}.{func.__module__}"
-        if task_type not in task_types:
-            raise TypeError(f'task_type is not supported please use any of [{",".join(task_types)}]')
         if name not in self.task_registry:
             self.task_registry[name] = func
         return self.task_registry[name]
 
-    def run(self, worker_type, workers):
+    def run(self, queue, workers):
```
Comment (arozumenko) [inline arbiter/__init__.py]: you have queue in self from Base class, why you need to pass it here?

File: arbiter/__init__.py [hunk 2]
```diff
@@ -138,12 +148,17 @@ def squad(self, tasks, callback=None):
         """
         Set of tasks that need to be executed together
         """
-        workers_count = {"heavy": 0, "light": 0}
+        workers_count = {}
         for each in tasks:
-            workers_count[each.worker_type] += each.tasks_count
+            workers_count[each.queue] = 0
```
Comment (arozumenko) [inline arbiter/__init__.py]: why you need 2 loops here
add each.queue if it is not there and then + each.tasks_count
single loop should be enough

**Review by arozumenko — CHANGES_REQUESTED** [review]

File: arbiter/__init__.py (hunk 1 above)
Comment (hunkom) [reply arbiter/__init__.py]: Fixed

File: arbiter/__init__.py (hunk 2 above)
Comment (hunkom) [reply arbiter/__init__.py]: Done

**Review by arozumenko — APPROVED** [review]

**Pull request merged**